bitbots_vision/bitbots_vision/vision_modules/yoeo/yoeo_handlers.py

```diff
@@ -501,10 +490,6 @@ def _assemble_full_path(cls, model_directory: str, subdir: Optional[str], filena
 
         return path
 
-    @classmethod
-    def get_names_file_path(cls, model_directory) -> str:
-        return cls._assemble_full_path(model_directory, None, "yoeo_names.yaml")
-
```

Comment:
This method is not needed anymore?

Reply:
That's correct. Loading the new model_config.yaml is solely handled by the ModelConfig class. Moreover, there is no "yoeo_names.yaml" anymore...